icu4j/main/tests/core/src/com/ibm/icu/dev/test/util/data/localeMatcherTest.txt

```diff
@@ -763,15 +763,15 @@ fr-FR >> fr
 ja-JP >> fr
 # For a language that doesn't match anything, return the default.
 zu >> en-GB
-root >> fr
```

Comment:
If these two cases failed because the test code is using forLanguageTag(), then please change to "und" not to "zxx". Or did it not work with "und"??

Reply:
I will take a closer look at that when I rebase my locale matcher branch.